active/ScrollingView/ScrollingView.md

```diff
@@ -0,0 +1,2584 @@
+    <mux:ItemsRepeater ItemsSource="{x:Bind Albums}" ItemTemplate="{StaticResource MyTemplate}">
+        <mux:ItemsRepeater.Layout>
+            <mux:UniformGridLayout RowSpacing="8" ColumnSpacing="8"/>
+        </mux:ItemsRepeater.Layout>
+    </mux:ItemsRepeater>
+</mux:ScrollingView>
+```
+
+**ScrollViewer markup (existing)**
+
+Identical to above.
+
+## Horizontal layout
+
+In this example the content is a StackPanel that's laying out its items horizontally. The `ScrollingView`'s ContentOrientation property is set to Horizontal to allow the user to scroll left and right to see all of the content.
+
+*`ScrollingView` (new)*
+
+```xml
+<mux:ScrollingView Width="500" Height="400"
+                  ContentOrientation="Horizontal">
+    <StackPanel Orientation="Horizontal">
+        <Button Width="300" Content="Button1"/>
+        <Button Width="300" Content="Button2"/>
+        <Button Width="300" Content="Button3"/>
+        <Button Width="300" Content="Button4"/>
+        <Button Width="300" Content="Button5"/>
+    </StackPanel>
+</mux:ScrollingView>
+```
+
+*ScrollViewer (existing)*
+
+```xml
+<ScrollViewer Width="500" Height="400"
+    HorizontalScrollBarVisibility="Auto" VerticalScrollBarVisibility="Disabled">
+    <StackPanel Orientation="Horizontal">
+        <Button Width="300" Content="Button1"/>
+        <Button Width="300" Content="Button2"/>
+        <Button Width="300" Content="Button3"/>
+        <Button Width="300" Content="Button4"/>
+        <Button Width="300" Content="Button5"/>
+    </StackPanel>
+</ScrollViewer>
+```
+
+## Two-dimensional (unconstrained) layout
+
+In this example the ContentOrientation is set to Both, indicating that the content can be scrolled both horizontally and vertically. The content's layout is unconstrained, meaning that it can be as wide and high as it wants to be, though the user will only be able to see what fits into the 500x400 viewport.
+
+The Image's natural size may be 2880 x 1800 and would be rendered using that size, but only 500x400 of the image would be displayed at a time.
+
+
+*`ScrollingView` (new)*
+
+```xml
+<mux:ScrollingView Width="500" Height="400" 
+                   ContentOrientation="Both">
+    <Image Source="Assets/LargeEiffelTower.png"/>
+</mux:ScrollingView>
+```
+
+*ScrollViewer (existing)*
+
+```xml
+<ScrollViewer Width="500" Height="400" HorizontalScrollBarVisibility="Auto">
+    <Image Source="Assets/LargeEiffelTower.png"/>
+</ScrollViewer>
+```
+
+## A Photo Viewer - Constrained to the Viewport Layout
+In this example the image will scale to fill the viewport while preserving its aspect ratio. 
+
+This is accomplished by:
+- Setting the ContentOrientation to None, indicating that the content should be constrained to fit within the `ScrollingView`'s space
+- Setting the Image's Stretch to Uniform, indicating that it should stretch up or down to fill its space without changing shape. 
+
+Since ZoomMode is enabled, the user can zoom in and then scroll.
```

Comment:
This paragraph is describing the particular scenario where the control is exactly sized to the natural size of the image. Until you zoom, there is no opportunity for scrolling since it fits perfectly. But after you zoom in, then it becomes possible to scroll because zooming makes the content's effective size bigger than the viewport.

Reply:
Say the natural image size is 5,000 x 4,000 but the Image element is set to 500 x 400 and in this "no scroll" mode. If I zoom in so that the extent is 5,000 x 4,000 will XAML show the 1:1 pixels of the original image or do I get the 500 x 400 image blown up?